docs/syntax_guide.rst

```diff
@@ -136,27 +141,33 @@ Properties added by `OrientedPoint`:
+ pitch [1]_           0              pitch in local coordinates
+ roll [1]_            0              roll in local coordinates
+ parentOrientation    global         basis for local coordinate system
+ viewAngles           (2π, π)        angles for visibility calculations
+ headingStdDev        5 degrees      mutation standard deviation for :prop:`heading`
 ===================  ==============  ================================================
 
 Properties added by `Object`:
 
-===================  ====================== ================================================
-   **Property**       **Default**                    **Meaning**
--------------------  ---------------------- ------------------------------------------------
- width                 1                    width of bounding box (X axis)
- length                1                    length of bounding box (Y axis)
- speed [1]_            0                    initial speed (later, instantaneous speed)
- velocity [1]_       from :prop:`speed`     initial velocity (later, instantaneous velocity)
- angularSpeed [1]_     0                    angular speed (change in heading/time)
- behavior              `None`               :term:`dynamic behavior`, if any
- allowCollisions      `False`               whether collisions are allowed
- requireVisible       `True`                whether object must be visible from ego
- regionContainedIn    `workspace`           Region the object must lie within
- cameraOffset          (0, 0)               position of camera for ‘can see’
-===================  ====================== ================================================
+=====================  ==================== ================================================
+   **Property**         **Default**                    **Meaning**
+---------------------  -------------------- ------------------------------------------------
+ width                 1                     width of bounding box (X axis)
+ length                1                     length of bounding box (Y axis)
+ height                1                     height of bounding box (Z axis)
+ shape                 `BoxShape`            shape of the object
+ speed [1]_            0                     initial (later, instantaneous) speed
+ velocity [1]_         from :prop:`speed`    initial (instantaneous) velocity
+ angularVelocity [1]_  (0, 0, 0)             initial (instantaneous) angular velocity
+ angularSpeed [1]_     0                     angular speed (change in :prop:`heading`/time)
+ behavior              `None`                :term:`dynamic behavior`, if any
+ allowCollisions       `False`               whether collisions are allowed
+ requireVisible        `False`               whether object must be visible from ego
+ regionContainedIn     `workspace`           `Region` the object must lie within
+ cameraOffset          (0, 0, 0)             position of camera for :keyword:`can see`
```

Comment:
Table is missing `baseOffset`, `contactTolerance`, etc.; can just put very brief descriptions here with a link to the full discussion, like for `cameraOffset`.